### ISSUE TITLE
@rancher/shell `0.3.3` causing errors with extensions

### ISSUE THREAD

**Issue by jordojordo** [rancher/dashboard#8152]
**Setup**
- Rancher version: `v2.7.1`
- Kubernetes version: `v1.24.10+k3s1`
- Browser type & version: FF `109.0.1`

**Describe the bug**

For reference: https://github.com/kubewarden/ui/issues/247

After bumping the `@rancher/shell` package to `0.3.3`, certain views from an extension remain blank with errors in the console. The errors include:
```console
TypeError: e.$plugin.getUIConfig is not a function
TypeError: e.extensionData is undefined
```

**To Reproduce**

- Install rancher on a k3s cluster
- Add the Kubewarden extension `1.0.1`
- Follow the installation process of the `kubewarden-controller`
- Add the `kubewarden-defaults` [helm-chart](https://github.com/kubewarden/helm-charts/tree/main/charts/kubewarden-defaults)
- Navigate to the Policy Server list view

**Result**
The page will fail to load and will only show a blank screen with the resource masthead

**Expected Result**

Should be able to view the Policy Server in the sortable table

**Screenshots**

___Policy server list view___
![ps](https://user-images.githubusercontent.com/40806497/218489533-4856d850-7a41-47e3-9701-8eb7283f232e.png)

___Cluster admission policy list view___
![cap](https://user-images.githubusercontent.com/40806497/218489636-463a12b0-ad66-4ff0-b23f-fd53b885160d.png)

___Errors in console___
![errors](https://user-images.githubusercontent.com/40806497/218489291-a475eb8d-189c-4156-8d29-19f50eeaab8e.png)

**Additional context**

These issues do __not__ appear when running Rancher within a Docker container.

**Comment by jordojordo** [comment]
Thanks @richard-cox, I didn't catch that PR before making the issue.

**Comment by jordojordo** [comment]
Closing as this has been fixed with the release of @rancher/shell `0.3.4`
